playground/Makefile

```diff
@@ -0,0 +1,19 @@
+.PHONY: build
+build:
+	podman build -f playground/Containerfile -t ghcr.io/redhat-et/playground --format docker playground
+
+models/llama-2-7b-chat.Q5_K_S.gguf:
+	wget -P models https://huggingface.co/TheBloke/Llama-2-7B-Chat-GGUF/resolve/main/llama-2-7b-chat.Q5_K_S.gguf -o /dev/null
+	# curl -s -S -L -f https://huggingface.co/TheBloke/Llama-2-7B-Chat-GGUF/resolve/main/llama-2-7b-chat.Q5_K_S.gguf -z $@ -o $@.tmp && mv -f $@.tmp $@ 2>/dev/null || rm -f $@.tmp $@
+
+.PHONY: install
+install:
+	pip install -r playground/tests/requirements.txt
```

Comment:
Hm, why do we need to reinstall here?  Isn't that done in the container image?  Or are there separate things?

Reply:
Those are sepearate things. This is the list of requirements the GHA runners should have to run pytest.